mlxtend/evaluate/per_class_accuracy.py

```diff
@@ -0,0 +1,43 @@
+# Sebastian Raschka 2014-2019
+# mlxtend Machine Learning Library Extensions
+#
+# A function for generating per class accuracy
+#
+# License: BSD 3 clause
+import numpy as np
+
+
+def per_class_accuracy(y_target, y_predicted, pos_label=None, binary=False):
+    """Compute per_class_accuracy for supervised learning.
+    Parameters
+    ------------
+    y_target : array-like, shape=[n_values]
+        True class labels or target values.
+    y_predicted : array-like, shape=[n_values]
+        Predicted class labels or target values.
+    pos_label : str or int, None by default.
+        The class whose accuracy score is to be reported.
+        'binary' has to be set to True.
+    binary : bool
```

Comment:
What happens if `binary` is set to `False`. Will this be a regular accuracy computation then? I think it looks like that. It would be nice to document that. 

I have another suggestion, though. How about 

1) calling this function just "accuracy" and computing the "average" per-class accuracy as an option?

E.g., instead of `binary`, we could use another parameter called `method`.

- `method='standard'` computes the regular accuracy
- `method='binary'` computes per-class accuracy for the positive class
- `method='average'` computes average per class accuracy similar to https://github.com/rasbt/mlxtend/blob/master/mlxtend/evaluate/scoring.py#L113

What do you think?

Reply:
Yes, that should have been documented. Thanks for pointing that out.

As for the suggestion, I think if you are looking to introduce modularity with respect to the individual metrics in "scoring.py", this would be a good first step. 
However, it would be interesting to know if you indeed want to eventually deprecate the current methodology of finding metrics and move towards standalone functions, like this?

Reply:
I think it would be good to support both, like scikit-learn does. The 'string' arguments in scoring are useful as a short-cut. However, it doesn't allow options like `pos_label`. Inside scoring, the idea would then be to use the `per_class_accuracy` function to implement the current `'average per-class accuracy'`

Reply:
So, implementation-wise, I think the API would be less confusing if we have just a single accuracy function that can compute the three types of metrics, based on the choice of "method". This however, means no separate per_class_accuracy method, as was originally intended.

Reply:
I agree. I think it's best if we call it just `classification_accuracy`? And then inside we have the `"per class"`  option

Reply:
i have to correct this, "`accuracy_score`" may be better because we also have `lift_score` already, and it is generally more consistent with sklearn then.